packages/plugin-feature/test/integration/spec/feature.js

```diff
@@ -45,4 +45,37 @@ describe(`plugin-feature`, function() {
       });
     });
   });
+
+  describe(`#setBatchUserFeatures()`, () => {
+    before(() => testUsers.create({count: 1})
+      .then((users) => {
+        spock = users[0];
+        spark = new CiscoSpark({
+          credentials: {
+            authorization: spock.token
+          }
+        });
+        return spark.device.register();
+      })
+    );
+
+    const featureUpdateArray = [{
+      key: `key1`,
+      val: `value1`,
+      type: `USER`,
+      mutable: `true`
+    }, {
+      key: `key2`,
+      val: `value2`,
+      type: `USER`,
+      mutable: `false`
+    }];
+
+    it(`sets a value for two user feature toggle`, () => {
+      return spark.feature.setBatchUserFeatures(featureUpdateArray)
```

Comment:
I don't think device.refresh gets called

    [`developer`, `entitlement`, `user`].forEach((collectionName) => {
      this.features.on(`change:${collectionName}`, (model, value, options) => {
        this.trigger(`change`, this, options);
        this.trigger(`change:features`, this, this.features, options);
      });
    });

I guess I can try to spy on change